manage.py

```diff
@@ -3,36 +3,37 @@
 from flask.cli import FlaskGroup
 
 from api import db
-from api.models import Task
-from api.utils import ToolhubClient
+from api.models import Field, Task
 from app import app
-from jobs.populate_db import insert_fields, populate_db
+from jobs.populate_db import insert_into_db, run_bulk_population_job
 
 cli = FlaskGroup(app)
-toolhub_client = ToolhubClient(app.config["TOOLHUB_API_ENDPOINT"])
 
 
 @cli.command("insert_fields")
-def run_field_insert():
-    """Inserts field data"""
-    insert_fields()
+def insert_fields():
+    """Insert data about annotations fields into the DB"""
+    BASE_DIR = app.config["BASE_DIR"]
+    with open(f"{BASE_DIR}/tests/fixtures/fields.json") as fields:
```

Comment:
* BASE_DIR is used in several places in this file, so it can be declared as a constant at the top
* In general, declaring all file paths at the top will make it easier to spot what needs changing if ever the current file and folder structure (or naming) changes in the future.
* It's good practice to wrap all path strings inside a pathlib.Path object

Reply:
Addressed with [2045ec3](https://github.com/wikimedia/toolhunt/pull/50/commits/2045ec3083eef8bb7f370bb58c111a25bb98413d)